### PR TITLE
media-gfx/openvdb: version bump to 4.0.2

### PR DESCRIPTION
**Pull request by dracwyrm**
Changed hardcoded paths to GNUInstallDirs
Closes Gentoo-bug: 611842

Patch to fixnumeric removal in Boost 1.65
Closes Gentoo-bug: 630740

Use PkgConfig to find OpenEXR and IlmBase

Possible fix for Gentoo-bug: 612942

Signed-off by: Jonathan Scruggs <j.scruggs@gmail.com>

**EDIT NOTE:** There is an external patch that needs to be transferred to dev space.

@SoapGentoo

### CODE REVIEW

**Comment by dracwyrm** [comment]
I added an external patch that needs to be transferred to dev space and the src uri updated in the ebuild.

File: media-gfx/openvdb/openvdb-4.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+SRC_URI="https://github.com/dreamworksanimation/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/dracwyrm/gentoo-patches/raw/master/${PN}/${P}-patchset-01.tar.xz"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+abi3-compat doc python"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="sys-libs/zlib
```
Comment (SoapGentoo) [inline media-gfx/openvdb/openvdb-4.0.2.ebuild]: `:=` now

Comment (dracwyrm) [reply media-gfx/openvdb/openvdb-4.0.2.ebuild]: Okay. It does get confusing on when to use that and when not.

File: media-gfx/openvdb/openvdb-4.0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit eutils flag-o-matic cmake-utils python-single-r1 versionator
```
Comment (SoapGentoo) [inline media-gfx/openvdb/openvdb-4.0.2.ebuild]: also, why do you still need `eutils`?

Comment (dracwyrm) [reply media-gfx/openvdb/openvdb-4.0.2.ebuild]: Um... I don't. XD

**Pull request closed without merging**